### PR TITLE
feature: Add searchbar with data from graphcool

### PR DESCRIPTION
**Pull request by SergeyMarynenko**

### CODE REVIEW

File: src/components/HomeScreen/index.js [hunk 1]
```diff
@@ -18,6 +14,11 @@ const TitleContainer = styled.View`
   padding: 10;
 `
 
+const ToolBarView = styled.ToolbarAndroid`
```
Comment (7s4r) [inline src/components/HomeScreen/index.js]: And for iOS?

Comment (SergeyMarynenko) [inline src/components/HomeScreen/index.js]: No for ios some UI elements need changes

File: src/components/SearchForm/index.js [hunk 2]
```diff
@@ -1,62 +1,161 @@
...
+  font-weight: 700;
+`
+
+class SearchForm extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      query: '',
+    }
+  }
```
Comment (7s4r) [inline src/components/SearchForm/index.js]: You can use the new method:
````
class SearchForm extends Component {
  state = {
    query: '',
  }
  ...
}
```

File: src/components/SearchForm/index.js [hunk 3]
```diff
@@ -1,62 +1,161 @@
...
+    }
+
+    if (error) {
+      return (
+        <Text>Error: {error.message}</Text>
+      )
+    }
+
+    const repairs = this.findRepair(this.state.query, allRepairs)
+
```
Comment (7s4r) [inline src/components/SearchForm/index.js]: Useless empty line

File: src/components/SearchForm/index.js [hunk 4]
```diff
@@ -1,62 +1,161 @@
...
+    const dataValues = repairs.length >= 1 && this.comp(this.state.query, repairs[0].name) ? [] : repairs
+
```
Comment (7s4r) [inline src/components/SearchForm/index.js]: Useless empty line

File: src/components/UserScreen/index.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      email: $email
+    ){
+      id
+    }
+  }`
+
+class UserScreen extends Component {
+  constructor(props) {
+    super(props)
+    this.lock = new Auth0Lock({ clientId: config.auth0.clientId, domain: config.auth0.domain })
```
Comment (7s4r) [inline src/components/UserScreen/index.js]: you can add this to componentWillMount

File: src/components/UserScreen/index.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+            variables: {
+              idToken: token.idToken,
+              name: profile.name,
+              email: profile.email,
+            },
+          })
+          ToastAndroid.show(translations('logedin'), ToastAndroid.SHORT)
+          Actions.pop()
+        } catch (e) {
+          ToastAndroid.show(translations('logedin'), ToastAndroid.SHORT)
```
Comment (7s4r) [inline src/components/UserScreen/index.js]: And for iOS?

File: src/components/HomeScreen/index.js (hunk 1 above)
Comment (7s4r) [inline src/components/HomeScreen/index.js]: Can we provide this now?

Comment (SergeyMarynenko) [inline src/components/HomeScreen/index.js]: No, only some Screens. I'm fix for ios when i will use mac os device

Comment (7s4r) [inline src/components/HomeScreen/index.js]: Ok

**Pull request merged**